DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateProgressController.cs

```diff
@@ -91,5 +94,25 @@ DelegateProgressAccessRoute accessedVia
 
             return RedirectToAction("Index", "ViewDelegate", new { formData.DelegateId });
         }
+
+        [HttpGet]
+        [Route("UnlockProgress")]
+        public IActionResult UnlockProgress(
+            int progressId,
+            int customisationId,
+            int delegateId,
+            DelegateProgressAccessRoute accessedVia
+        )
+        {
+            progressService.UnlockProgress(progressId);
+            notificationService.SendUnlockRequest(progressId);

```

Comment:
There is a requirement to ensure the link sent out in the notification email directs the user to the this refactored site rather than the old site if the TrackingSystem feature flag is on (i.e. TrackingSystem feature is enabled).
I don't see any changes made related to this.

Reply:
The url is read from the database. This value is read from ConfigName column which is in config table 
SELECT * FROM Config WHERE ConfigName = 'TrackingSystemBaseURL'
We need to update the url here. I have updated the ticket

Reply:
So the requirement here would be to check the feature manager to see if the tracking system is enabled. 
If tracking is enabled then use the new site base url in the email.
If tracking is disabled then use the old site base url in the email as before.
This should not require a DB change.
We either use the old value from the DB/AppSettings.
Or the new url from AppSettings.
The DigitalLearningSolutions.Data.Helpers.ConfigHelper will be of use here.

Reply:
Added a check as suggested. This will set url based on tracking system 
Please ensure you have config settings in the database (in config table)

![image](https://user-images.githubusercontent.com/90783892/145002146-02ae3ba6-a31b-4f16-9cbd-1adea7244353.png)

Reply:
@showkath10 Can you verify that this value exists in all the required databases.
And/or write a migration script that ensures it is populated correctly.

Reply:
I've added this migration